src/Workspaces/Core/Portable/LinkedFileDiffMerging/LinkedFileDiffMergingLogger.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Internal.Log;
+using static Microsoft.CodeAnalysis.LinkedFileDiffMergingSession;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal class LinkedFileDiffMergingLogger
+    {
+        private const string Id = nameof(Id);
```

Comment:
Its used as the key of a key value pair on line [107](https://github.com/dotnet/roslyn/pull/9479/files#diff-687368884bfe3f281361c29ae2832b2dR107) and [137](https://github.com/dotnet/roslyn/pull/9479/files#diff-687368884bfe3f281361c29ae2832b2dR137).  The common pattern appears to be to use nameof for these string.